ext/native-decls/MumbleSetServerAddress.md

```diff
@@ -0,0 +1,13 @@
+---
+ns: CFX
+apiset: client
+---
+## MUMBLE_SET_SERVER_ADDRESS
+
+```c
+void MUMBLE_SET_SERVER_ADDRESS(char* address, int port);
+```
+
```

Comment:
Another description missing, such as for instance 'Changes the Mumble server address to connect to, and reconnects to the new address.'